Release pipeline errors caused by AADAuthenticationMethodPolicy.

### Description of the issue

I built my environment according to the Azure DevOps whitepaper.
And I added AzureAD workload using exported settings.
Error occurred in the release pipeline.
If I comment out AADAuthenticationMethodPolicyFido2, the error did not occur.
What is causing it?　And how should I do to solve these?

### Microsoft 365 DSC Version

V1.23.1213.1

### Which workloads are affected

Azure Active Directory

### The DSC configuration

```powershell
AADAuthenticationMethodPolicyFido2 "AADAuthenticationMethodPolicyFido2"
        {
            Credential                       = $Credential;
            Ensure                           = "Present";
            Id                               = "Fido2";
            IncludeTargets                   = @(
                MSFT_AADAuthenticationMethodPolicyFido2IncludeTarget{
                    Id = 'all_users'
                    TargetType = 'group'
                }
            );
            IsAttestationEnforced            = $False;
            IsSelfServiceRegistrationAllowed = $True;
            KeyRestrictions                  = MSFT_MicrosoftGraphfido2KeyRestrictions{
                IsEnforced = $False
                EnforcementType = 'block'
                AaGuids = @()
            };
            State                            = "disabled";
        }
```


### Verbose logs showing the problem

```powershell
##[error]ClassVersion Qualifier not found.
    + CategoryInfo          : ObjectNotFound: (root/Microsoft/...gurationManager:String) [], CimException

    + FullyQualifiedErrorId : MI RESULT 6

    + PSComputerName        : localhost
```


### Environment Information + PowerShell Version

_No response_

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,12 @@
 # Change log for Microsoft365DSC
 
+# UNRELEASED
+
+* AADAuthenticationMethodPolicyFido2
+  * Add missing class identifier to schema.
+    FIXES [#4900](https://github.com/microsoft/Microsoft365DSC/issues/4900)
+    FIXES [#4079](https://github.com/microsoft/Microsoft365DSC/issues/4079)
+
 # 1.24.724.1
 
 * IntuneAntivirusPolicyWindows10SettingCatalog
@@ -9,7 +16,7 @@
     FIXES [#3966](https://github.com/microsoft/Microsoft365DSC/issues/3966)
 * IntuneEndpointDetectionAndResponsePolicyLinux
   * Initial release.
-* IntuneEndointDetectionAndResponsePolicyMacOS
+* IntuneEndpointDetectionAndResponsePolicyMacOS
   * Initial release.
 * IntuneWindowsUpdateForBusinessFeatureUpdateProfileWindows10
   * Introduces new properties and updates the handling of the

--- a/Modules/Microsoft365DSC/DSCResources/MSFT_AADAuthenticationMethodPolicyFido2/MSFT_AADAuthenticationMethodPolicyFido2.schema.mof
+++ b/Modules/Microsoft365DSC/DSCResources/MSFT_AADAuthenticationMethodPolicyFido2/MSFT_AADAuthenticationMethodPolicyFido2.schema.mof
@@ -11,7 +11,7 @@ class MSFT_AADAuthenticationMethodPolicyFido2ExcludeTarget
     [Write, Description("The object identifier of an Azure AD group.")] String Id;
     [Write, Description("The type of the authentication method target. Possible values are: group and unknownFutureValue."), ValueMap{"user","group","unknownFutureValue"}, Values{"user","group","unknownFutureValue"}] String TargetType;
 };
-
+[ClassVersion("1.0.0")]
 class MSFT_AADAuthenticationMethodPolicyFido2IncludeTarget
 {
     [Write, Description("The object identifier of an Azure AD group.")] String Id;

